aws/secrets.js

```diff
@@ -32,52 +32,47 @@ const initialiseSecrets = async (region, wallet) => {
         ENCRYPTION_PRIVATE_KEY: mySecrets.ENCRYPTION_PRIVATE_KEY,
         ENCRYPTION_PUBLIC_KEY: mySecrets.ENCRYPTION_PUBLIC_KEY,
         ENCRYPTION_PASSPHRASE: mySecrets.ENCRYPTION_PASSPHRASE,
+        DATA_ENGINEERING_ACCESS_KEY: mySecrets.DATA_ENGINEERING_ACCESS_KEY,
+        DATA_ENGINEERING_SECRET_KEY: mySecrets.DATA_ENGINEERING_SECRET_KEY,
       };
     }
   } catch (err) {
     console.error('Failed to load AWS secrets: ', err);
   }
 };
 
+const getSecret = (secretName) => {
+  if (myLocalSecrets === null) throw new Error('Unknown secrets');
+  if (!myLocalSecrets[secretName]) throw new Error(`Unknown ${secretName} secret`);
+
+  return myLocalSecrets[secretName];
+}
+
 const encryptionPrivate = () => {
-  if (myLocalSecrets !== null) {
-    if (!myLocalSecrets.ENCRYPTION_PRIVATE_KEY) {
-      throw new Error('Unknown ENCRYPTION_PRIVATE_KEY secret');
-    } else {
-      return myLocalSecrets.ENCRYPTION_PRIVATE_KEY;
-    }
-  } else {
-    throw new Error('Unknown secrets');
-  }
+  return getSecret('ENCRYPTION_PRIVATE_KEY');
 };
 
 const encryptionPublic = () => {
-  if (myLocalSecrets !== null) {
-    if (!myLocalSecrets.ENCRYPTION_PUBLIC_KEY) {
-      throw new Error('Unknown ENCRYPTION_PUBLIC_KEY secret');
-    } else {
-      return myLocalSecrets.ENCRYPTION_PUBLIC_KEY;
-    }
-  } else {
-    throw new Error('Unknown secrets');
-  }
+  return getSecret('ENCRYPTION_PUBLIC_KEY');
 };
 
 const encryptionPassphrase = () => {
-  if (myLocalSecrets !== null) {
-    if (!myLocalSecrets.ENCRYPTION_PASSPHRASE) {
-      throw new Error('Unknown ENCRYPTION_PASSPHRASE secret');
-    } else {
-      return myLocalSecrets.ENCRYPTION_PASSPHRASE;
-    }
-  } else {
-    throw new Error('Unknown secrets');
-  }
+  return getSecret('ENCRYPTION_PASSPHRASE');
+};
+
+const dataEngineeringAccessKey = () => {
+  return getSecret('DATA_ENGINEERING_ACCESS_KEY');
+};
```

Comment:
This can be shortened:

```suggestion
const dataEngineeringAccessKey = () => getSecret('DATA_ENGINEERING_ACCESS_KEY');
```

Reply:
Same with the others too